app/main.py

```diff
@@ -1,3 +1,62 @@
+from __future__ import annotations
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[int, Distance]) -> Distance:
+        if isinstance(other, (int, float)):
```

Comment:
what logic?) you are checking other on a float and int type, but you expect only int)